packages/core/src/components/fields/SchemaField.tsx

```diff
@@ -199,7 +200,9 @@ function SchemaFieldRender<T, F>(props: FieldProps<T, F>) {
   if (wasPropertyKeyModified) {
     label = name;
   } else {
-    label = uiOptions.title || props.schema.title || schema.title || name;
+    label = schema.hasOwnProperty(ADDITIONAL_PROPERTY_FLAG)
```

Comment:
Does this work with `ADDITIONAL_PROPERTY_FLAG  in schema`? I ask because eslint complains about the `hasOwnProperty()` function and we've removed them from everywhere else in the code